src/freeze-dry/inline-styles.js

```diff
@@ -44,7 +44,7 @@ async function inlineLinkedStylesheets({rootElement, docUrl}) {
         } catch (err) {
             stylesheetText = '/* Oops! Freeze-dry failed to save this stylesheet. */'
         }
-        const styleEl = rootElement.ownerDocument.createElement('style')
+        const styleEl = rootElement.createElement('style')
```

Comment:
I kept encountering an error with this exact statement, which I did not before it was changed. So I started fiddling with the code to hack my way through, but no change in the testing code made it run. If there are any suggestions, I will be extremely happy to hear them.